docs/index.html

```diff
@@ -42,6 +42,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Appreciates code reviews from their peers that are constructive and helpful.</li>
         <li>Is open to learning.</li>
         <li>Asks for help when they need it.</li>
+        <li>Listen to diffuse conflict and formulate better solutions.</li>
```

Comment:
For verb consistency! 😄 

```suggestion
        <li>Listens to diffuse conflict and formulate better solutions.</li>
```

Reply:
+1 to this. If you could merge @daneah's changes that would be swell.